app/controllers/roadmap_controller.rb

```diff
@@ -2,6 +2,6 @@ class RoadmapController < ApplicationController
   include Cacheable
 
   def index
-    render locals: { full_width: true }
+    slimmer_template "gem_layout"
```

Comment:
Because this is using the constrained width layout, we can remove the `<div class="govuk-width-container">` in the [roadmap view](https://github.com/alphagov/frontend/blob/478cc123326fa5d25b814bd7cc4102a270c7259a/app/views/roadmap/index.html.erb#L10).